src/wfgoto.cpp

```diff
@@ -267,13 +267,14 @@ vector<wstring> SplitIntoWords(LPCTSTR szText)
 {
 	vector<wstring> words;
 
-	wstringstream ss;
-	ss.str(szText);
-	wstring item;
-	while (getline(ss, item, L' '))
+	wchar_t tempStr[MAXPATHLEN];
+	wcscpy_s(tempStr, szText);
+	PWCHAR token{ nullptr };
+	PWCHAR word = wcstok_s(tempStr, szPunctuation, &token);
+	while (word)
 	{
-		if (item.size() != 0)
-			words.push_back(item);
+		words.push_back(word);

```

Comment:
Seems to me we should retain the check on word.size() since it appears wcstok_s will return an empty string if two punctuation are in a row.

Reply:
wcstok_s() doesn't deliver empty strings. Tried this with e.g. 
`SplitIntoWords(L"_-_test-_-foo_-_");`
It returns 'test' and 'foo'